nimare/dataset.py

```diff
@@ -103,6 +104,17 @@ def __init__(self, source, target="mni152_2mm", mask=None):
         self.texts = _dict_to_df(id_df, data, key="text")
         self.basepath = None
 
+        if "z_stat" in self.coordinates.columns:
+            # "z_stat" column may contain Nones
+            if not self.coordinates["z_stat"].isna().any():
+                # Raise warning if coordinates dataset contains both positive and negative z_stats
+                if ((self.coordinates["z_stat"].values >= 0).any()) and (
+                    (self.coordinates["z_stat"].values < 0).any()
+                ):
+                    warnings.warn(
+                        "Coordinates dataset contains both positive and negative z_stats"
```

Comment:
I think we should expand the warning a bit, to give users some context. Perhaps something like "The algorithms currently implemented in NiMARE are designed for one-sided tests, so results may not make sense."

@jdkent WDYT?